### PR TITLE
Conference on front

### PR DESCRIPTION
**Pull request by jospoortvliet**
Put the conference on the front page & link to the Friday Focus blog on the conf page.

### CODE REVIEW

**Comment by jospoortvliet** [comment]
@MariusBluem @NinaNC if you could have a look and spot any big issues... would rock. If OK, update branch & merge pls ;-)

File: page-homepage.php [hunk 1]
```diff
@@ -12,9 +12,9 @@
 	<div class="container">
 		<div class="row">
 			<div class="col-md-6 toptext ">
-				<a class="announcement" href="https://nextcloud.com/blog/welcome-to-nextcloud-12/">
+				<a class="announcement" href="https://nextcloud.com/conf/">
 				<span class="type">Fresh</span>
```
Comment (MariusBluem) [inline page-homepage.php]: Call it Event instead of Fresh and change the color to (I would say red) please 🙈

Comment (Espina2) [inline page-homepage.php]: @MariusBluem why red? Orange is complementary color of blue so let it orange. Don't use no sense colors if you have no explanation for that.

Comment (Espina2) [inline page-homepage.php]: Anyway I agree with the text

**Comment by jospoortvliet** [comment]
OK?

**Comment by MariusBluem** [comment]
:+1: Content wise 😏

**Pull request merged**